R/data_interface.R

```diff
@@ -702,13 +723,16 @@ spark_read_source <- function(sc,
 #' \dontrun{
 #' sc <- spark_connect(
 #'   master = "local",
-#'   config = list(sparklyr.shell.packages = "mysql:mysql-connector-java:5.1.44")
+#'   config = list(
+#'     `sparklyr.shell.driver-class-path` = "/usr/share/java/mysql-connector-java-8.0.25.jar"
+#'   )
 #' )
 #' spark_write_jdbc(
 #'   sdf_len(sc, 10),
 #'   name = "my_sql_table",
 #'   options = list(
 #'     url = "jdbc:mysql://localhost:3306/my_sql_schema",
+#'     driver = "com.mysql.jdbc.Driver",
```

Comment:
When I tried running this use case it did not work unless I have the `driver = <class name>` specification.